operator/pkg/controlplane/apiserver/manifests.go

```diff
@@ -182,6 +183,10 @@ spec:
         - --audit-log-path=-
         - --audit-log-maxage=0
         - --audit-log-maxbackup=0
+        ports:
+          - containerPort: 8080
+            name: metrics
+            protocol: TCP
```

Comment:
Is it mandatory to expose the port explicitly? 

Echo from [comments of Ports](https://github.com/kubernetes/kubernetes/blob/16e8911fdccbdb348a0b3291acf6102494494f9b/staging/src/k8s.io/api/core/v1/types.go#L2738-L2741)
> 	// List of ports to expose from the container. Not specifying a port here
	// DOES NOT prevent that port from being exposed. Any port which is
	// listening on the default "0.0.0.0" address inside a container will be
	// accessible from the network.

My concern here is that the `--metrics-bind-address` [might be customized](https://github.com/karmada-io/karmada/blob/b192c9c0d3fd3c3bc584528b9dc52cee9ac593ba/operator/pkg/apis/operator/v1alpha1/type.go#L375-L389), but no way to update the port accordingly.

Reply:
Thanks @RainbowMango 
Remove port

Reply:
Seems not

Reply:
Or extract ports settings into helm value?